examples/MagicWeather/yarn.lock

```diff
@@ -16,51 +26,51 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@babel/compat-data@npm:^7.27.2, @babel/compat-data@npm:^7.27.7, @babel/compat-data@npm:^7.28.5":
-  version: 7.28.5
-  resolution: "@babel/compat-data@npm:7.28.5"
-  checksum: d7bcb3ee713752dc27b89800bfb39f9ac5f3edc46b4f5bb9906e1fe6b6110c7b245dd502602ea66f93790480c228605e9a601f27c07016f24b56772e97bedbdf
+"@babel/compat-data@npm:^7.22.6, @babel/compat-data@npm:^7.26.8":
```

Comment:
Hmm interesting that some of these ended up being downgrades... Not a blocker though.